README.md

```diff
@@ -211,6 +211,36 @@ USB device into the same physical USB port (this method is also used in Linux ke
 Linux USB permissions
 =====================
 
+Kernels after 6.0
+-----------------
+
+On Linux Kernel 6.0 and later (e.g. `uname -r` shows a version numer greater or equal to 6.0) there is
+a standard interface to turn USB hub ports on or off, that can be used instead of libusb.
```

Comment:
Just move this to start on Linux permissions paragraph to say something like this:
```
On Linux Kernel 6.0 and later (e.g. `uname -r` shows a version numer greater or equal to 6.0) there is
a standard interface to turn USB hub ports on or off, and `uhubctl` will try using it instead of `libusb`
if it is supported by current kernel.
Rules below have slight differences specific to Linux 6.x, but you can have both enabled
if you are not sure which kernel version you have or will have.
```

Reply:
numer => number